### ISSUE TITLE
[explicit-member-accessibility] "explicit" option suggestions

### ISSUE THREAD

**Issue by alumni** [typescript-eslint/typescript-eslint#4647]
- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/explicit-member-accessibility": "error"
  }
}
```

```TS
export class Dog {
    bark() {
        console.log('woof!');
    }
}
```

**Expected Result**

The method `bark()` is missing the accessibility modified. `eslint --fix` should add `public`.

**Actual Result**

The method `bark()` is missing the accessibility modified. `eslint --fix` cannot automatically fix this.

**Additional Info**

The docs https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md#attributes mention this rule is fixable. This only works for `{ accessibility: 'no-public' }` but not for `{ accessibility: 'explicit' }` (which is the default).

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.14.0` |
| `@typescript-eslint/parser`        | `5.14.0` |
| `TypeScript`                       | `4.6.2` |
| `ESLint`                           | `8.10.0` |
| `node`                             | `16.14.0` |

**Comment by bradzacher** [comment]
This is by design.
The rule is not intended to automatically add public everywhere for you - it is intended to make you think about what accessibility you should be using for your members.

Is public really the right accessibility to use for the member you just wrote?

If we simply autofixed every case to public then it would defeat the purpose of the rule.

**Comment by JoshuaKGoldberg** [comment]
Reopening per #5492 to track that we'd accept a _suggestion_ fixer.
